rs/tests/nested/hostos_upgrade.rs

```diff
@@ -119,7 +119,8 @@ pub fn upgrade_hostos(env: TestEnv) {
     .unwrap();
 
     info!(logger, "Waiting for Orchestrator dashboard...");
-    host.await_orchestrator_dashboard_accessible().unwrap();
+    host.await_orchestrator_dashboard_accessible()
+        .unwrap_or_else(|e| try_logging_guestos_diagnostics(&host, &logger, e));
```

Comment:
Also following https://github.com/dfinity/ic/pull/7495, it might be easier to read with the diagnostics in line:
```
if let Err(err) = host.await_orchestrator_dashboard_accessible() {
  // error log
  // list of `block_on_bash_script_and_log`, or util func
  // panic
}
```